provisioner/roles/manage_ec2_instances/templates/etchosts/etchosts_rhel.j2

```diff
@@ -21,6 +21,6 @@
 
 {% for vm in ansible_node_facts.instances %}
 {% if 'student' + item == vm.tags.Student %}
-{{ vm.public_ip_address }} {{vm.tags.Student}}.{{ec2_name_prefix|lower}}.{{workshop_dns_zone}} {{ vm.tags.short_name }}
+{{ vm.private_ip_address }} {{vm.tags.Student}}.{{ec2_name_prefix|lower}}.{{workshop_dns_zone}} {{ vm.tags.short_name }}
```

Comment:
So, with this the nodes don't have public IPs anymore, am I right? We need confirmation for each workshop individually if the exercises really work with private IPs.

Reply:
And we should get away from using the IPs in favor of using the (/etc/hosts) names anyway... ;)